src/main/java/org/takes/rq/RqHref.java

```diff
@@ -67,16 +79,36 @@
         public Base(final Request req) {
             super(req);
         }
+
+        // @todo #445 RqMethod already validates Request-Line and
```

Comment:
@Happy-Neko the puzzle format should be like `#445:30min/DEV`
otherwise rultor won't merge the PR